src/ADAL.NET/AuthenticationContext.cs

```diff
@@ -771,11 +790,12 @@ public AuthenticationResult AcquireTokenSilent(string resource, ClientAssertion
         /// <param name="redirectUri">Address to return to upon receiving a response from the authority.</param>
         /// <param name="userId">Identifier of the user token is requested for. This parameter can be <see cref="UserIdentifier"/>.Any.</param>
         /// <param name="extraQueryParameters">This parameter will be appended as is to the query string in the HTTP authentication request to the authority. The parameter can be null.</param>
+        /// <param name="claims">Additional claims that are needed for authentication. Acquired from the AdalClaimChallengeException. This parameter can be null.</param>
         /// <returns>URL of the authorize endpoint including the query parameters.</returns>
-        public Uri GetAuthorizationRequestURL(string resource, string clientId, Uri redirectUri, UserIdentifier userId, string extraQueryParameters)
+        public Uri GetAuthorizationRequestURL(string resource, string clientId, Uri redirectUri, UserIdentifier userId, string extraQueryParameters, string claims)
```

Comment:
This is a public API change. Make sure you add either a default value for the new `claims` parameter here, or better add an overload taking the original 5 parameters).

NB: this issue also exists in the v3 version of ADAL but we missed it! 😢